jest.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  collectCoverage: true, //process.env.NODE_ENV === 'production' ? true : false,
+  collectCoverage: process.env.NODE_ENV === 'production' ? true : false,
```

Comment:
tests doesnt run in `production` so this will not collect the coverage. lets set it back to `true`